### DISCUSSION TITLE
leaflet map

### DISCUSSION THREAD

**Discussion by gdauby** [dreamRs/conrappli#7] [Ideas]
Voici un exemple de leaflet que j'utilise, avec le mode `leaflet::markerClusterOptions()`

Il me semble que lorsque des points sont exactement superposés, ils sont 'éclatés' lorsqu'on clique dessus.

Il y a aussi les 3 `leaflet::providers` qui sont utiles.

Le `leafpop::popupTable` devrait idéalement afficher les variables sélectionnées.

`leaflet::leaflet() %>%
          leaflet::addProviderTiles(leaflet::providers$OpenStreetMap, 
                                    group =  "OSM") %>%
          leaflet::addProviderTiles(leaflet::providers$Esri.WorldImagery, 
                                    group =  "Esri")%>%
          leaflet::addProviderTiles(leaflet::providers$OpenTopoMap, group =  "Open Topo Map")%>%
          leaflet::addMarkers(
            data = res_selected_sf,
            lng = ~ ddlon,
            lat = ~ ddlat,
            popup = leafpop::popupTable(
              res_selected_sf
                )
            ),
            clusterOptions = leaflet::markerClusterOptions()
          ) %>% 
          leaflet::addLayersControl(
            baseGroups = c("OSM", "Esri", "Open Topo Map"),
            overlayGroups = c("Georeferenced collections"),
            options = leaflet::layersControlOptions(collapsed = FALSE)
          ) %>% 
          leafem::addMouseCoordinates(epsg = NULL, proj4string = NULL,
                              native.crs = FALSE)`

**Comment by pvictor** [comment]
Effectivement, je ne connaissais pas cette fonctionnalité.
Par contre cela ne fonctionne via le module de sélection de crosstalk (https://github.com/rstudio/leaflet/issues/478), on va devoir faire un entre-deux.
